modcc/symge.cpp

```diff
@@ -78,30 +85,48 @@ void gj_reduce(sym_matrix& A, symbol_table& table) {
 
     auto define_sym = [&table](symbol_term_diff t) { return table.define(t); };
 
-    std::vector<unsigned> pivots;
-    for (unsigned r = 0; r<A.nrow(); ++r) {
-        pivots.push_back(r);
-    }
+    auto get_pivots = [&A](std::vector<unsigned> remaining_rows) {
```

Comment:
Pass by reference to avoid a copy, i.e. `const std::vector<unsigned>&`.

Reply:
Done